src/components/session/SortOptions/index.tsx

```
@@ -0,0 +1,30 @@
+
+export default function SortOptions({ value, onChange }: SortOptionsProps) {
+  return (
+    <Dropdown size="lg">
+      <Dropdown.Trigger className="bg-transparent">
+        {getOptionLabel(SORT_OPTIONS, value)}
+      </Dropdown.Trigger>
+      <Dropdown.Content>
+        {SORT_OPTIONS.map(({ label, value }) => (
+          <Dropdown.Item
+            key={value}
+            selected={value === value}
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)